arviz/stats/stats.py

```diff
@@ -12,9 +12,8 @@
 from scipy.optimize import minimize
 import xarray as xr
 
+from ..plots.plot_utils import _fast_kde, get_bins
 from ..data import convert_to_inference_data, convert_to_dataset, InferenceData, CoordSpec, DimSpec
-from ..plots.kdeplot import _fast_kde
-from ..plots.plot_utils import get_bins
 from .diagnostics import _multichain_statistics, _mc_error, ess, _circular_standard_deviation
 from .stats_utils import (
     make_ufunc as _make_ufunc,
```

Comment:
below on line 27, histogram is imported from stats_utils in a weird way, it can be moved to this import.

I think this triggers some kind of circular import and is the root of the import problems

Reply:
@OriolAbril I noticed this problem earlier too and has removed the import problem with `b2f600d` commit.

Reply:
Yes, but the code should use the `histogram` function, not rewrite it here from scratch.